data-analysis/template.yaml

```diff
@@ -201,6 +201,21 @@ Resources:
             MetricName: ZendeskWebhookApiWafIpReputationListMetric
             SampledRequestsEnabled: true
 
+  AuditDataRequestQueue:
```

Comment:
Can we shove the word Events in the id and the name? Just so it's clear what it's for should anyone come across it with less context than us :)

Reply:
Yep, done.